policy/PfsMapper.yaml

```diff
@@ -376,6 +376,24 @@ datasets:
     tables:
     - raw
 
+  dmQaResidualImage:
+    template: 'dmQaResidualImage/%(dateObs)s/v%(visit)06d/dmQaResidualImage-%(visit)06d-%(arm)s%(spectrograph)d.pdf'
+    python: object
+    persistable: ignored
+    storage: MatplotlibStorage
+    level: Ccd
+    tables:
+    - raw
+
+  dmQaResidualStats_pickle:
+    template: 'dmQaResidualImage/%(dateObs)s/v%(visit)06d/dmQaResidualStats-%(visit)06d-%(arm)s%(spectrograph)d.pickle'
```

Comment:
I've changed the output name to have the word `stats` in it but this still goes to the `dmQaResidualImage` base in the template, is this okay @PaulPrice ?

Reply:
That's fine.

You can probably drop the `_pickle` in the name, now it's unique.

Reply:
Should I make the base of the template match? I want to think of this as a file structure and would almost put the base of both templates to be `dmQaResidual`, which would then have the plots and stats in them.

Reply:
> Should I make the base of the template match? I want to think of this as a file structure and would almost put the base of both templates to be `dmQaResidual`, which would then have the plots and stats in them.

I just got this working locally and it does put that in a directory structure in the rerun, so I think I'd like to name the base something more generic, like `detectorMapQA` and then put all assets from the DM QA in there. Let me know if that's not the correct idea.

Reply:
Sounds good to me.